index.html

```diff
@@ -9,26 +9,183 @@
   </head>
   <body class="bg">
   <div class="wrapper">
-    <header class="header">
-      <a class="mylogo">My Logo</a>
-      <nav class='nav'>
-        <img src='images/hamburg.svg' alt='' title=''>
-      </nav>
-    </header>
-    <section>
-     	<h1 class="myheader">I’m PRATAP Glad to see you!</h1>
-    	<h4 class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
-    	<a class="connect">LET’S CONNECT</a>
-	<ul class="social-media">
-
-	<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
-	  
-	</ul>
-    </section>
+    <div class='mainsection'>
+	    <header class="header">
+	      <a class="mylogo">PRATAP PANABAKA</a>
+	      <nav class='nav'>
+		<img src='images/hamburg.svg' alt="Click for the Menu Options" title=''>
+	      </nav>
+	    </header>
+	    <section>
+	     	<p class="myheader">I’m PRATAP Glad to see you!</p>
+	    	<p class="content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+	    	<h3 class="connect">LET’S CONNECT</h3>
+		<ul class="social-media">
+
+		<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='20' height='20' alt="Twitter Profile" title=''></a></li>
+		<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt="LinkedIn Profile" title=''></a></li>
+		<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
+		<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt="GitHub Profile" title=''></a></li>
+		<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
+		  
+		</ul>
+	    </section>
+    </div>
+    <div class='projects-section'>
+    <div class="project-wrapper grid-box">
```

Comment:
- [ ] Kindly note that the container for the whole my works section is supposed to be a `<section>` and not a `<div>`. `<section>` is a semantically better way to markup your code.

- For a visual of what I mean:
 ```
<section class="project-wrapper grid-box">
 <!-- Your code here -->
</section>
```